tox.ini

```diff
@@ -224,11 +216,6 @@ commands =
     py27-profile: python -m tests.profiling.run pytest --capture=no --verbose --benchmark-disable --ignore-glob="*asyncio*" {posargs} tests/profiling
     # Coverage is excluded from profile because of an issue with Python 3.5.
     py3{5,6,7,8,9,10}-profile: python -m tests.profiling.run pytest --no-cov --capture=no --verbose --benchmark-disable {posargs} tests/profiling
-# run only the opentrace tests
-    opentracer: python -m pytest {posargs} tests/opentracer/core
-    opentracer_asyncio: python -m pytest {posargs} tests/opentracer/test_tracer_asyncio.py
-    opentracer_tornado-tornado{40,41,42,43,44}: python -m pytest {posargs} tests/opentracer/test_tracer_tornado.py
```

Comment:
looks like we weren't running the tornado tests with 5.x, 6.x or latest like it seemed from above on line 66

Reply:
the tests are actually broken in these versions: 
```
.riot/venv_py3510_mock_pytest700_pytest-mock_coverage_pytest-cov_opentracing_hypothesis/lib/python3.5/site-packages/opentracing/scope_managers/tornado.py:24: in <module>
    import tornado.stack_context
E   ImportError: No module named 'tornado.stack_context'
```